trinity/main.py

```diff
@@ -205,7 +169,16 @@ class DBManager(BaseManager):
 
 
 @with_queued_logging
-def run_lightnode_process(chain_config: ChainConfig) -> None:
+def launch_node(chain_config: ChainConfig) -> None:
+    display_launch_logs(chain_config)
+
+    node_class = choose_node_class(chain_config)
```

Comment:
Maybe condense to `chain_config.get_node()`.  The `choose_node_class` looks a lot like something that the `ChainConfig` class should handle (though it may need to leave the `utils` namespace soon since `utils` is not supposed to import from outside of `utils`.)

Reply:
#759 - moving ChainConfig and adding the node class lookup

I'll also see if it makes sense to separate out another PR with the Node refactor that lets you initialize with only the chain config. I think that might be too tied into the rest of this PR to make sense, though.